processor/src/main/java/org/mapstruct/ap/internal/processor/AnnotationBasedComponentModelProcessor.java

```diff
@@ -50,8 +55,11 @@
     public Mapper process(ProcessorContext context, TypeElement mapperTypeElement, Mapper mapper) {
         this.typeFactory = context.getTypeFactory();
 
-        String componentModel = MapperConfiguration.getInstanceOn( mapperTypeElement )
-                                                   .componentModel( context.getOptions() );
+        String componentModel =
+            MapperConfiguration.getInstanceOn( mapperTypeElement ).componentModel( context.getOptions() );
```

Comment:
You can extract `MapperConfiguration.getInstanceOn( mapperTypeElement )` into a variable and use it instead of creating it twice